### DISCUSSION TITLE
pdfcompare

### DISCUSSION THREAD

**Discussion by hporeddy32** [red6/pdfcompare#90] [Q&A]
Hi,

With red6/pdfcompare is it possible to exclude words/text. As of now i see you can exclude using pixel co-ordinates.
Any help would be greatly appreciated

**Comment by finsterwalder** [comment]
PdfCompare works only pixel based. It does not read text from the Pdf and as such your request is not doable.
You can only exlude regions of a page.

**Comment by gastendonk** [comment]
> Any help would be greatly appreciated

It's easy to define exclusions visually with this tool.

**Reply by hporeddy32** [reply]
Thank you :) 
Can I do not overlay the result, instead display it side by side ?
And if i am running from Intellij , how can i specify the exclusion file in the UI , i only see expected and actual pdf selection option

**Reply by gastendonk** [reply]
from memory...

If comparing two PDF files the produced diff file contains both files in red and green.

Start PDF Compare Studio:
This is our batch file: java -DDPI=150 -classpath pdfcompare-1.1.55-SNAPSHOT-full.jar de.redsix.pdfcompare.ui.DisplayMain %1 %2 %3

There's an Exclusions Button that shows a sidebar to manage the exclusions.

I guess a PDF Compare Studio documentation is missing !?

**Reply by finsterwalder** [reply]
You can use the UI to create the proper exclusion file and then store that file and specify it with a "withExclusion(...)" method call.

If you don't want red and green changes in one output file, you would have to run two compares using different color settings.
